app/services/metrc_service/plant/start.rb

```diff
@@ -51,7 +48,9 @@ def build_start_payload
       end
 
       def create_plantings_from_package
-        call_metrc(:create_plantings_package, create_package_plantings_payload)
+        call_metrc(:create_plantings_package, create_plantings_from_package_payload)
+
+        MetrcService::Plant::Move.call(@ctx, @integration) if seeding_unit.item_tracking_method != 'none'
```

Comment:
add a comment here explaining what this means... something like "if customers start their batches already in flowering mode, then we need to report a 'move' to metrc right after starting the batch."